index.html

```diff
@@ -191,7 +191,36 @@ <h4 class="skillTitle"><span class="cir"></span> Skills</h4>
                     </ul>
                 </li>
             </ul>
-         </section> 
+        </section>
+
+        <section id="contact-section">
+            <h2 id="contact-title">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h2>
+            <form action="/my-handling-form-page" method="post" id="contactForm">
+              
+                <label>
+                    <input type="text" id="name-contact" class="inputContact" name="user_name" placeholder="Full name" maxlength="30"/>
+                </label>
+                <label>
+                    <input type="email" id="mail-contact" class="inputContact" name="user_email" placeholder="Email address"/>
+                </label>
+                <label>
+                    <textarea id="msg-contact" class="inputContact" name="user_message" placeholder="Enter text here" maxlength="500"></textarea>
+                </label>
+                <input type="submit" name="submit" value="Get in touch" class="submit-contact orangeBtn"/>
+            </form>
```

Comment:
kindly all label should be required. the user could not submit the form if the whole form was not filled in  please [read the project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)